### PR TITLE
Add use-cases first draft

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

**Review by mestery — APPROVED** [review]
Seems reasonable, though I still wonder if boards without a BMC are actually DPU/IPUs vs. just being SmartNICs.

**Comment by stolsma** [comment]
> Seems reasonable, though I still wonder if boards without a BMC are actually DPU/IPUs vs. just being SmartNICs.

@mestery What is defined as a BMC here? Is that a separate "processor env" that is controlling the hw env of the DPU/IPU next to the processor complex which will run the IPU/DPU OS? If so, existence of a BMC is in my opinion not required for a DPU/IPU implementation as it could also be implemented in the mother chassis where the IPU/DPU is placed in (just my 2 cents...)

**Comment by glimchb** [comment]
> > Seems reasonable, though I still wonder if boards without a BMC are actually DPU/IPUs vs. just being SmartNICs.
> 
> @mestery What is defined as a BMC here? Is that a separate "processor env" that is controlling the hw env of the DPU/IPU next to the processor complex which will run the IPU/DPU OS? If so, existence of a BMC is in my opinion not required for a DPU/IPU implementation as it could also be implemented in the mother chassis where the IPU/DPU is placed in (just my 2 cents...)

I know RH require to have DPU bmc in their DPU/IPU deployments, for example

**Comment by stolsma** [comment]
> to



> > > Seems reasonable, though I still wonder if boards without a BMC are actually DPU/IPUs vs. just being SmartNICs.
> > 
> > 
> > @mestery What is defined as a BMC here? Is that a separate "processor env" that is controlling the hw env of the DPU/IPU next to the processor complex which will run the IPU/DPU OS? If so, existence of a BMC is in my opinion not required for a DPU/IPU implementation as it could also be implemented in the mother chassis where the IPU/DPU is placed in (just my 2 cents...)
> 
> @stolsma, I know RH require to have DPU bmc in their DPU/IPU deployments, for example

Wow, for me as a OS/software solution customer that requirement would be a big negative as it would hamper the flexibility I want to create by using a DPU/IPU solution...

**Comment by stolsma** [comment]
> > > > Seems reasonable, though I still wonder if boards without a BMC are actually DPU/IPUs vs. just being SmartNICs.
> > > 
> > > 
> > > @mestery What is defined as a BMC here? Is that a separate "processor env" that is controlling the hw env of the DPU/IPU next to the processor complex which will run the IPU/DPU OS? If so, existence of a BMC is in my opinion not required for a DPU/IPU implementation as it could also be implemented in the mother chassis where the IPU/DPU is placed in (just my 2 cents...)
> > 
> > 
> > @stolsma, I know RH require to have DPU bmc in their DPU/IPU deployments, for example
> 
> I'm generally saying that the complex is one thing which separates a smartNIC from a DPU/IPU.

Ok, we agree again! 😉

File: USECASES.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+               sZTP – when used?
+               FDO – when used? 
+- Telemetry for service deployment
+- Secure wipe? (reprovisioning) 
+               What does it erase
+                              What memory devices
+               Leave UEFI
+                              What is present on a “bare metal” DPU
+
+- Authenticating DHCP –
```
Comment (glimchb) [inline USECASES.md:45]: some envs are not allowed DHCP

File: USECASES.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # Deployment scenarios
 
 - Private network/cloud – (DHCP/DNS trusted)
-               Telco
+               Telco Core
+               Telco Near Edge
+               Telco Far Edge
                Enterprise
+               Industrial Edge (DPUs not currently prevalent)
```
[the remark this answers is not in the document]
Comment (glimchb) [reply USECASES.md:8]: @tedstreete please comment

**Pull request merged**